testnets/saagetestnet/assetlist.json

```diff
@@ -1,24 +1,24 @@
 {
     "$schema": "../../assetlist.schema.json",
-    "chain_id": "saage-internal-testnet-1.3",
+    "chain_name": "saagetestnet",
     "assets": [
       {
         "description": "The SGE token is primarily a governance token for the Saage chain.",
         "denom_units": [
           {
-            "denom": "usaage",
+            "denom": "usge",
             "exponent": 0,
             "aliases": []
           },
           {
-            "denom": "saage",
+            "denom": "sge",
             "exponent": 6,
             "aliases": []
           }
         ],
-        "base": "usaage",
+        "base": "usge",
         "name": "Saage",
-        "display": "saage",
+        "display": "sge",
         "symbol": "SGE",
         "logo_URIs": {
           "png": "https://defitech-logo.s3.amazonaws.com/Final+Logo+Saage.png",
```

Comment:
These images are chain logos, not token logos. Chain logos can be added to the chain.json. Token logos should be added here. Check fails because these images need to be uploaded to the Chain Registry